.circleci/config.yml

```diff
@@ -74,7 +88,7 @@ jobs:
 
       - run:
           name: Run tests
-          command: pytest -k "not examples" --cov=prefect .
+          command: pytest -rfEsx --skip-formatting --cov=prefect .
```

Comment:
What is `-rfEsx`?

Reply:
It adds additional "reason" reporting for failed, xfailed, erors, and skipped tests.  So, for example, if I run `pytest -rfEsx` on master right now, at the end of the test report I now see:
```
SKIP [1] tests/cli/test_cli.py:28: CLI not yet fully implemented
SKIP [1] tests/environments/storage/test_docker_storage.py:146: Needs to be mocked so it can work on CircleCI
SKIP [1] tests/environments/storage/test_docker_storage.py:166: Needs to be mocked so it can work on CircleCI
SKIP [10] /Users/chris/Developer/prefect/tests/conftest.py:102: Airflow tests skipped by default unless --airflow flag provided to pytest.
XFAIL tests/engine/cloud/test_cloud_flows.py::test_deep_map_with_a_retry
  Sometimes fails due to some shared-state error
```
This allows us to manually check sometimes that doc checks are running (and not skipped), etc.